packages/mendel-core/trees.js

```diff
@@ -42,7 +42,7 @@ MendelTrees.prototype.findTreeForVariations = function(bundle, variations) {
     return finder.found();
 };
 
-MendelTrees.prototype.findServerVariationMap = function(variations) {
+MendelTrees.prototype.findServerVariationMap = function(variations, bundleList) {
```

Comment:
If you take a look at the other methods of this class, many methods receive a singe bundleId as first param and variations as second param. Can we change the param order?

```js
MendelTrees.prototype.findServerVariationMap = function(bundle, variations) {
```

Do it need it to be a list? I don't know if we really need it to be a list. If we need, it's ok, otherwise we could just pass a single ID and the API would be very similar to the other functions, right?